### ISSUE TITLE
XY SCRIPT

### ISSUE THREAD

**Issue by cardenluo** [LucianoCirino/efficiency-nodes-comfyui#21]
Script generated preview images cannot preview multiple images at the same time, previous versions were able to do so in blender -comfyui

**Comment by LucianoCirino** [comment]
Can you show me a workflow example of this error please?

**Comment by cardenluo** [comment]
![Snipaste_2023-06-12_20-51-57](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/133877069/80380398-88ca-4808-aa00-c38b238ecd46)

**Comment by LucianoCirino** [comment]
Great, thanks for the clarification. 

The normal behavior of the output image for the KSampler(Efficient) after doing an XY Plot script was changed to the individual generated plot images rather than the plot result (I am not sure why your UI is showing one image, in comfyui it shows all the images). In the very latest update however I added an option for you to be able to change it so the KSampler(Efficient) outputs the plot image. In the 'node_settings.json' change the xy_plot_as_output_image to "true" as seen below.
![image](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/112517630/3ba15921-b690-4f9a-9be7-96981ac388c0)
